datalad/cmdline/common_args.py

```diff
@@ -50,7 +50,7 @@
     'pbs-runner', ('--pbs-runner',),
     dict(choices=['condor'],
          default=None,
-         help="""execute command by scheduling it via available PBS.  For settings, config file will be consulted""")
+         help="""DEPRECATED, will be removed in a future release.""")
```

Comment:
useful would have been to add a comment with version (0.16) when it is ok to remove, or someone would need to do archaeological expedition to figure out for sure if it is ok time to remove

don't want to hold this PR with this, so let's proceed

Reply:
Filed https://github.com/datalad/datalad/issues/5962 with milestone 0.16